docs/DeveloperGuide.adoc

```diff
@@ -881,26 +881,31 @@ image::DoneSessionSequenceDiagram.png[]
 
 1. The `LogicManager` uses the `TaTrackerParser` to first parse the user command.
 
-2. The `TaTrackerParser` sees that the command is a type *sessions* and passes the command
+2. The `TaTrackerParser` sees that the command is a *sessions* type and passes the command
```

Comment:
Markdown, not bold. 

... sees that the command is a `SessionDone` command and ...